src/implementations/twig/components/fact-figures/fact-figures.story.js

```diff
@@ -18,7 +18,7 @@ const viewAll = { ...data3Col.view_all };
 const getArgTypes = (data) => {
   return {
     icon: {
-      defaultValue: data.items[0].icon.name,
+      defaultValue: system === 'ec' ? 'digital' : data.items[0].icon.name,
```

Comment:
why do we force the use of this specific icon?

Reply:
On EC version, they are using the same digital icon everywhere, on EU they are using a different icon for each items.

Reply:
I don't think we need to keep this difference, that's just an example.
We could use the same content for EC and EU (including the icons)